code/datums/components/crafting/recipes.dm

```diff
@@ -12,7 +12,7 @@
 	var/list/tool_behaviors
 	/// Type paths of items needed but not consumed. Lazy list.
 	var/list/tool_paths
-	var/time = 30 //time in deciseconds
+	var/time = 3 SECONDS //time in deciseconds
```

Comment:
Make sure you clean up comments like this. It's no longer deciseconds!

Reply:
You got it.